docs/settings.rst

```diff
@@ -839,6 +839,12 @@ the ``TAG_FEED_ATOM`` and ``TAG_FEED_RSS`` settings:
    Maximum number of items allowed in a feed. Feed item quantity is
    unrestricted by default.
 
+.. data:: RSS_FEED_SUMMARY_ONLY = True
+
+   Include only summary in the <description/> tag of RSS feeds items. When
```

Comment:
I also found that backticked tag name w/o angle brackets used in the TYPOGRIFY_IGNORE_TAGS description. So I’m going to do the same to keep consistency.